### PR TITLE
add missing code contributors to all contributors in README and `.all-contributors.rc` file

### PR DESCRIPTION
**Pull request by gr2m**
addresses https://github.com/processing/p5.js/issues/6337#issuecomment-1681460654

rendered preview of new README:
https://github.com/processing/p5.js/tree/add-missing-code-contributors#contributors

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -1354,9 +1354,7 @@
       "avatar_url": "https://avatars2.githubusercontent.com/u/15272015?v=4",
       "profile": "https://github.com/adilrabbani",
       "contributions": [
-        "code",
-        "bug",
-        "example"
+        "code"
```
Comment (gr2m) [inline .all-contributorsrc:1357]:
```suggestion
        "code",
        "bug",
        "example"
```

File: .all-contributorsrc [hunk 2]
```diff
@@ -1389,9 +1387,7 @@
       "avatar_url": "https://avatars1.githubusercontent.com/u/6486359?s=460&v=4",
       "profile": "https://github.com/AidanNelson",
       "contributions": [
-        "bug",
-        "code",
-        "doc"
+        "code"
```
Comment (gr2m) [inline .all-contributorsrc:1390]:
```suggestion
        "code",
        "bug",
        "example"
```

File: README.md [hunk 3]
```diff
@@ -340,12 +340,12 @@ We recognize all types of contributions. This project follows the [all-contribut
...
       <td align="center" valign="top" width="16.66%"><a href="https://github.com/yutinglu413"><img src="https://avatars1.githubusercontent.com/u/25344311?v=4?s=120" width="120px;" alt="Yuting Lu"/><br /><sub><b>Yuting Lu</b></sub></a><br /><a href="https://github.com/processing/p5.js/commits?author=yutinglu413" title="Documentation">📖</a></td>
-      <td align="center" valign="top" width="16.66%"><a href="https://github.com/adilrabbani"><img src="https://avatars2.githubusercontent.com/u/15272015?v=4?s=120" width="120px;" alt="Adil Rabbani"/><br /><sub><b>Adil Rabbani</b></sub></a><br /><a href="https://github.com/processing/p5.js/commits?author=adilrabbani" title="Code">💻</a> <a href="https://github.com/processing/p5.js/issues?q=author%3Aadilrabbani" title="Bug reports">🐛</a> <a href="#example-adilrabbani" title="Examples">💡</a></td>
+      <td align="center" valign="top" width="16.66%"><a href="https://github.com/adilrabbani"><img src="https://avatars2.githubusercontent.com/u/15272015?v=4?s=120" width="120px;" alt="Adil Rabbani"/><br /><sub><b>Adil Rabbani</b></sub></a><br /><a href="https://github.com/processing/p5.js/commits?author=adilrabbani" title="Code">💻</a></td>
```
Comment (gr2m) [inline README.md:343]:
```suggestion
      <td align="center" valign="top" width="16.66%"><a href="https://github.com/adilrabbani"><img src="https://avatars2.githubusercontent.com/u/15272015?v=4?s=120" width="120px;" alt="Adil Rabbani"/><br /><sub><b>Adil Rabbani</b></sub></a><br /><a href="https://github.com/processing/p5.js/commits?author=adilrabbani" title="Code">💻</a> <a href="https://github.com/processing/p5.js/issues?q=author%3Aadilrabbani" title="Bug reports">🐛</a> <a href="#example-adilrabbani" title="Examples">💡</a></td>
```

File: README.md [hunk 4]
```diff
@@ -340,12 +340,12 @@ We recognize all types of contributions. This project follows the [all-contribut
...
-      <td align="center" valign="top" width="16.66%"><a href="https://github.com/AidanNelson"><img src="https://avatars1.githubusercontent.com/u/6486359?s=460&v=4?s=120" width="120px;" alt="Aidan Nelson"/><br /><sub><b>Aidan Nelson</b></sub></a><br /><a href="https://github.com/processing/p5.js/issues?q=author%3Aaidannelson" title="Bug reports">🐛</a> <a href="https://github.com/processing/p5.js/commits?author=aidannelson" title="Code">💻</a> <a href="https://github.com/processing/p5.js/commits?author=aidannelson" title="Documentation">📖</a></td>
+      <td align="center" valign="top" width="16.66%"><a href="https://github.com/AidanNelson"><img src="https://avatars1.githubusercontent.com/u/6486359?s=460&v=4?s=120" width="120px;" alt="Aidan Nelson"/><br /><sub><b>Aidan Nelson</b></sub></a><br /><a href="https://github.com/processing/p5.js/commits?author=aidannelson" title="Code">💻</a></td>
```
Comment (gr2m) [inline README.md:348]:
```suggestion
      <td align="center" valign="top" width="16.66%"><a href="https://github.com/AidanNelson"><img src="https://avatars1.githubusercontent.com/u/6486359?s=460&v=4?s=120" width="120px;" alt="Aidan Nelson"/><br /><sub><b>Aidan Nelson</b></sub></a><br /><a href="https://github.com/processing/p5.js/issues?q=author%3Aaidannelson" title="Bug reports">🐛</a> <a href="https://github.com/processing/p5.js/commits?author=aidannelson" title="Code">💻</a> <a href="https://github.com/processing/p5.js/commits?author=aidannelson" title="Documentation">📖</a></td>
```

**Review by gr2m — COMMENTED** [review]
revert incorrect changes (the CLI did that, it shouldn't have 🤷🏼)

File: .all-contributorsrc [hunk 5]
```diff
@@ -1389,9 +1389,9 @@
       "avatar_url": "https://avatars1.githubusercontent.com/u/6486359?s=460&v=4",
       "profile": "https://github.com/AidanNelson",
       "contributions": [
-        "bug",
         "code",
-        "doc"
+        "bug",
+        "example"
```
Comment (gr2m) [inline .all-contributorsrc:1394]: sorry copy & paste fail

```suggestion
        "bug",
        "code",
        "doc"
```

File: .all-contributorsrc [hunk 6]
```diff
@@ -1389,6 +1389,7 @@
       "avatar_url": "https://avatars1.githubusercontent.com/u/6486359?s=460&v=4",
       "profile": "https://github.com/AidanNelson",
       "contributions": [
+        "code",
```
Comment (gr2m) [inline .all-contributorsrc:1392]:
```suggestion
```

**Comment by Qianqianye** [comment]
Thank you so much @gr2m!

**Pull request merged**